.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+  "presets": [
+    "env",
+    "react",
+    "stage-0"
+  ],
+  "plugins": [
+    "transform-class-properties",
+    "transform-decorators",
+    "transform-react-constant-elements",
+    "transform-react-inline-elements"
+  ]
+}
```

Comment:
@jorgebucaran @mindplay-dk It seems the terser CLI adds default babelrc with above presets and plugins. However having an empty babelrc also works for our purpose. I am more inclined to keep this file but would be interested in hearing your thoughts.

Reply:
Assuming that's the case, does anyone know why terser adds it?

Reply:
On further investigation, terser package is ok. But, babel-env has a postinstall script https://github.com/epsitec-sa/babel-env/blob/master/package.json#L9 which creates a default babelrc. This feature is also documented in README.md for this package https://github.com/epsitec-sa/babel-env#babel-env

![image](https://user-images.githubusercontent.com/802242/50264494-d816d780-03cf-11e9-806b-f79dec6209c1.png)

Reply:
Currently we need it for jest. Without env, jest would complain for not finding   env preset. Alternative would be to add these packages:
```
yarn add --dev babel-jest babel-core@^7.0.0-bridge.0 @babel/core regenerator-runtime
```

Reply:
@osdevisnot Yep, thanks for clarifying it.

Reply:
Right, thanks for clarifying that.